app/assets/stylesheets/styles.scss

```diff
@@ -323,4 +323,13 @@ pzsh-topbar {
 .center-xy{
   align-items: center;
   justify-content: center;
-}
\ No newline at end of file
+}
+
+.margin-row {
+  margin-top: 7.5rem;
```

Comment:
Ins't there a bootstrap class for this? Or does it need to be this specific?

Reply:
Deleted because no usage found.